README.md

```diff
@@ -6,39 +6,39 @@ Problem statement:
 4) Once both of them approve, notification are sent to insurance admin.
 5) Insurance admin can check for approvals of hospital and lab after which he will calculate the claim amount and do the claim.
 
- `HealthCare.sol` contract maintains the logic for this DApp.  
+ `HealthCare.sol` contract maintains the logic for this DApp.
   The web pages found in the `Web-client` folder are used to communicate with the deployed smart contract and also allow logging in for each specific user
 
 ## Steps to deploy and interact with the contract:
 1. Copy and paste the contract code on https://remix.ethereum.org/
-2. Run an instance of ganache-cli on your local machine and connect your metamask wallet to it. Also, add the first 3 accounts from ganache to your metamask by importing their private keys and assign the following names to it:  
-    account 1: Hospital admin  
-    account 2: Lab admin  
-    account 3: Patient  
-3. Replace the Lab admin's address inside the constructor on line 27 with the Lab admin's address that we configured in the previous step.
+2. Run an instance of ganache-cli on your local machine and connect your metamask wallet to it. Also, add the first 3 accounts from ganache to your metamask by importing their private keys and assign the following names to it:
+    account 1: Hospital admin
+    account 2: Lab admin
+    account 3: Patient
+3. Replace the Lab admin's address [here in file : 2_deploy_HealthCare.js](https://github.com/Rishabh42/HealthCare-Insurance-Ethereum/blob/60792a7d275e84cca7aa3a6d4734554cdbd67283/migrations/2_deploy_HealthCare.js#L2) with the Lab admin's address that we configured in the previous step.
```

Comment:
These steps are for deploying on Remix, so it should be that the user should pass an address from Ganache (which is different from the hospital admin's address) to the constructor while deploying the contract.

Reply:
Pass The Lab admin's address (From ganache) in constructor [here in file : 2_deploy_HealthCare.js](https://github.com/Rishabh42/HealthCare-Insurance-Ethereum/blob/60792a7d275e84cca7aa3a6d4734554cdbd67283/migrations/2_deploy_HealthCare.js#L2).

-- this works ?